### PR TITLE
exporter: add metadata distributions for opensearch exporter

### PR DESCRIPTION
**Pull request by YANG-DB**
**Description:** 
Adding metadata distributions for opensearch exporter

```
type: opensearch

status:
  class: exporter
  stability:
    alpha: [traces]
    development: [logs]
  distributions: [contrib, aws]
  codeowners:
    active: [Aneurysm9, MitchellGale, MaxKsyunz, YANG-DB]
```

### CODE REVIEW

File: exporter/opensearchexporter/metadata.yaml [hunk 1]
```diff
@@ -5,5 +5,6 @@ status:
   stability:
     alpha: [traces]
     development: [logs]
+  distributions: [contrib, aws]
```
Comment (Aneurysm9) [inline exporter/opensearchexporter/metadata.yaml:8]:
```suggestion
  distributions: [contrib]
```

This exporter is not included in ADOT at this time.

**Comment by atoulme** [comment]
Please run `make generate` and `make gendistributions` as well to generate the README and distribution report changes.

**Review by atoulme — APPROVED** [review]

**Comment by YANG-DB** [comment]
Hi @Aneurysm9  - can u plz review ?

**Review by Aneurysm9 — APPROVED** [review]

**Comment by bogdandrutu** [comment]
@atoulme  please also make sure tests run for PRs that you mark ready for merge.

**Comment by atoulme** [comment]
Good to go here @bogdandrutu - @YANG-DB please refrain from rebasing or updating the PR after it's marked ready to merge, so we don't have to rerun tests for you.

**Pull request merged**